packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2866,12 +2956,42 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
   RenderRepaintBoundary createRenderObject(BuildContext context) => new RenderRepaintBoundary();
 }
 
+/// Is invisible during hit testing.
+///
+/// When [ignoring] is `true`, this widget (and its subtree) is invisible
+/// to hit testing. It still consumes space during layout and paints its child
+/// as usual. It just cannot be the target of located events because it returns
```

Comment:
Crazy.  http://www.quickanddirtytips.com/education/grammar/when-to-use-a-comma-before-because